File: web-app/src/types/workingTime.ts
import { DayOfWeek } from "./commonTypes";

export interface WorkingTimeData { 
    id: string; 
    day: DayOfWeek; 
    fromTime: string; 
    toTime: string; 
    isActive: boolean; 
    createDate?: string; 
    createdBy?: string; 
}

File: API/Entities/WorkingTime.cs

    namespace API.Entities
    {
        public class WorkingTime
        {
            public Guid Id { get; set; }
            public DayOfWeek Day { get; set; }
            public TimeOnly FromTime { get; set; }
            public TimeOnly ToTime { get; set; }

            public bool IsActive { get; set; } = true;
            public DateTime? CreateDate { get; set; }
            public string? CreatedById { get; set; }
            public AppUser? CreatedBy { get; set; }
        }
    }